WooCommerce/Resources/Info.plist

```diff
@@ -61,6 +61,12 @@
 	<string>To take photos or videos to add to your Products or support tickets.</string>
 	<key>NSPhotoLibraryUsageDescription</key>
 	<string>To save photos from camera for Product images, or to add photos or videos to your Products or support tickets.</string>
+	<key>NSLocationWhenInUseUsageDescription</key>
+	<string>Location access is required in order to accept payments.</string>
+	<key>NSBluetoothPeripheralUsageDescription</key>
+	<string>Bluetooth access is required in order to connect to supported card readers.</string>
+	<key>NSBluetoothAlwaysUsageDescription</key>
+	<string>This app uses Bluetooth to connect to supported card readers.</string>	
```

Comment:
> cc @AliSoftware . Once we merge this, we might need to update the privacy recipe in the App Store listing.

I'm guessing those are the new things that might warrant the update you're talking about? 🙃 

I'll take a look at the Privacy Labels to see if those would be affected by these changes, thanks for the heads up (and kudos for thinking about those)! 👍 

---

Might be worth writing a short P2 once this PR gets merged (i.e a "Platform9 Request") to "officially" request those Privacy Label changes though, both to keep track and inform of the request in a more publicly-visible space, and also because we will soon rotate release managers – so better be sure the next RM for Woo will see the info too. 🙇

Reply:
That's correct. Those are the new things that might trigger the need for an update to the privacy labels.

> Might be worth writing a short P2 once this PR gets merged (i.e a "Platform9 Request") to "officially" request those Privacy Label changes though,

Absolutely, good point. We'll do that as soon as we merge. 

We also have a p2 ready to go with a checklist for undoing the integration if for some reason we get rejected

Reply:
Wow perfect! 👌 
![image](https://media0.giphy.com/media/Ezw8AgcLYKayI/giphy.gif?cid=790b761122076119e84ae781c3105edc4c091b812d99a3da&rid=giphy.gif&ct=g)